spec/lib/services/sku-pack-service-spec.js

```diff
@@ -107,8 +117,22 @@ describe("SKU Pack Service", function() {
                 destroy: sinon.stub()
             };
 
+            waterline.templates = {
```

Comment:
I checked these stub for waterline service has not been restored, so this may impact other unit test.